hera_qm/xrfi.py

```diff
@@ -1373,9 +1373,10 @@ def chi_sq_pipe(uv, alg='zscore_full_array', modified=False, sig_init=6.0,
 #############################################################################
 
 
-def xrfi_run(ocalfits_file, acalfits_file, model_file, data_file, history,
+def xrfi_run(ocalfits_file=None, acalfits_file=None, model_file=None, data_file=None, history=None,
```

Comment:
For abscal and omnical, we need to be able to turn on/off gain-based flags and chi^2-based flags (both the med/meanfilt ones and the absolute ones using the chi_sq_pipe). For H3C/H4C, I think we'll probably want to turn off abscal chi^2 flagging, since it might get triggered when the RIMEz model fails.